### PR TITLE
FlowForge 1.6 release post

### PR DESCRIPTION
**Pull request by iskerrett**
Release post for FlowForge 1.6

 - [x ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2023/04/flowforge-1-6-released.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+PostHog events & groups first pass [#1922](https://github.com/flowforge/flowforge/pull/1922)
+
+
+Add Algolia Search in Docs  [#620](https://github.com/flowforge/website/pull/620) 
+
+
+## Bug Fixes
+
+Delelting instances from the instance list fails [#1859](https://github.com/flowforge/flowforge/issues/1859)
```
Comment (robmarcer) [inline src/blog/2023/04/flowforge-1-6-released.md:44]: Spelling error - Delelting

**Review by robmarcer — CHANGES_REQUESTED** [review]

**Comment by robmarcer** [comment]
Some of the improvements maybe should have better titles, the issue titles are often written without consideration to how they would read in release notes.

File: src/blog/2023/04/flowforge-1-6-released.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Update email address verification  [#813](https://github.com/flowforge/flowforge/issues/813)
+
+Reminder email about trial doesn't include a link to FF Cloud [#1815](https://github.com/flowforge/flowforge/issues/1815)
+
+Sign-up coupons improvement [#1788](https://github.com/flowforge/flowforge/issues/1788)
+
+Deprecate FF_PROJECT_* envvars and add FF_INSTANCE_* equivalents [#1844](https://github.com/flowforge/flowforge/issues/1844)
+
+PostHog events & groups first pass [#1922](https://github.com/flowforge/flowforge/pull/1922)
```
Comment (MarianRaphael) [inline src/blog/2023/04/flowforge-1-6-released.md:36]: I would change the title to "Improved PostHog integration"

File: src/blog/2023/04/flowforge-1-6-released.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+## Other Improvements
+
+
+Update email address verification  [#813](https://github.com/flowforge/flowforge/issues/813)
+
+Reminder email about trial doesn't include a link to FF Cloud [#1815](https://github.com/flowforge/flowforge/issues/1815)
+
+Sign-up coupons improvement [#1788](https://github.com/flowforge/flowforge/issues/1788)
+
+Deprecate FF_PROJECT_* envvars and add FF_INSTANCE_* equivalents [#1844](https://github.com/flowforge/flowforge/issues/1844)
```
Comment (MarianRaphael) [inline src/blog/2023/04/flowforge-1-6-released.md:34]: I would change the title to: Terminology change for environment variables.

File: src/blog/2023/04/flowforge-1-6-released.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+PostHog events & groups first pass [#1922](https://github.com/flowforge/flowforge/pull/1922)
+
+Add Algolia Search in Docs  [#620](https://github.com/flowforge/website/pull/620)
```
Comment (MarianRaphael) [inline src/blog/2023/04/flowforge-1-6-released.md:38]: Change to: "Introducing search bar to the docs/handbook page"

**Review by MarianRaphael — APPROVED** [review]

File: src/blog/2023/04/flowforge-1-6-released.md (hunk 2 above)
Comment (iskerrett) [reply src/blog/2023/04/flowforge-1-6-released.md]: I already changed to Integrate with PostHog events.

File: src/blog/2023/04/flowforge-1-6-released.md (hunk 3 above)
Comment (iskerrett) [reply src/blog/2023/04/flowforge-1-6-released.md]: I spoke with Rob about this and we agreed to break it out to two descriptions.

File: src/blog/2023/04/flowforge-1-6-released.md (hunk 4 above)
Comment (iskerrett) [reply src/blog/2023/04/flowforge-1-6-released.md:40]: done

**Review by robmarcer — APPROVED** [review]

**Pull request merged**